src/cpr_sdk/cli/search.py

```diff
@@ -190,7 +190,7 @@ def main(
 
                 table.add_row(
                     text,
-                    str(round(hit.relevance, SCORES_NUM_DECIMALS)),
+                    str(round((hit.relevance or 0.0), SCORES_NUM_DECIMALS)),
```

Comment:
this is just for printing, so how about

```suggestion
                    str(round(hit.relevance, SCORES_NUM_DECIMALS)) if hit.relevance is not None else "n/a",
```

Reply:
Looks good!